### PR TITLE
[RFR] Improved “Closure” entry

### PR DESCRIPTION
**Pull request by robbestad**

### CODE REVIEW

File: glossary/CLOSURE.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
 # Closure
 
-A closure is a function value created from a nested function declaration or function expression (i.e. lambda expression) whose body contains one or more references to variables declared in an outer (but not global) scope. Closures are functions that refer to independent variables. Therefore private variables can be made possible with closures.
+A closure is function that closes over its environment. It has access to the state of the environment, but the variables inside the closure is private.
```
Comment (KittyGiraudel) [inline glossary/CLOSURE.md]: are private\* :)

File: glossary/CLOSURE.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+
+Example:
+
+    (function (){
```
Comment (KittyGiraudel) [inline glossary/CLOSURE.md]: Let’s used fenced code blocks with `js` syntax to have colors. :)

Comment (robbestad) [inline glossary/CLOSURE.md]: Ah, TIL!

File: glossary/CLOSURE.md (hunk 1 above)
Comment (robbestad) [inline glossary/CLOSURE.md]: Thanks :)

**Comment by KittyGiraudel** [comment]
Great! :)

**Pull request merged**